### PR TITLE
Video list page

### PR DESCRIPTION
**Pull request by foyzulkarim**

### CODE REVIEW

File: docs/notes.txt [hunk 1]
```diff
@@ -0,0 +1,36 @@
+uploadedBy: User
+Video:
```
Comment (foyzulk2023) [inline docs/notes.txt:2]: this is a test comment

Comment (foyzulkarim) [reply docs/notes.txt:2]: thanks for the test comment. 💯

Comment (imamhossainpro) [reply docs/notes.txt:2]: Hi there.

**Review by imamhossainpro — COMMENTED** [review]
Good

**Pull request merged**